setup.py

```diff
@@ -1,8 +1,13 @@
+from os import sys, path
+
 from skbuild import setup
 
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from itkVersion import VERSION
```

Comment:
Indeed, when I mentioned `within the itk package`, I meant:

* `itkVersion.py` remains in the `ITKPythonPackage`
* `itk/__init__.py` provided by ITK is updated to do:

```
try:
    from . import itkVersion
    setattr(thisModule, '__version__', itkVersion.VERSION)
except ImportError:
    pass
```


and optionally `itkVersion.py` could be moved from `ITKPythonPackage/itkVersion.py` to `ITKPythonPackage/itk/itkVersion.py` but that wouldn't bring much ..